contents/handbook/people/hiring-process.md

```diff
@@ -0,0 +1,82 @@
+
+## Advertising the job
+
+There are a couple of places that are worth posting to:
+
+- HackerNews Who's Hiring
+  - Tend to get high quality candidates, and people interested in working at startups. 
+  - See [Tim's comment history](https://news.ycombinator.com/threads?id=timgl) for a template
+- RemoteOK
+  - High volume of candidates, but much lower quality.
+- AngelList
+  - We found Eric through there. Higher quality than RemoteOK and pretty high volume.
+
+## Interviewing
+
+The stages of interviewing are:
+
+- Application
+- Interview with Tim
+- Interview with James
+- Interview with someone from the team
+- 3 days of contracting work
+- Decision
+
+### Application
+
+It's worth reading applications and resume carefully. For current roles, the things we look for are
+
+- Experience with relevant technologies (Python or similar, React or similar, something to do with big data is a bonus)
+- Has started a project from scratch, without outside help
+  - Usually this manifests as having been the founder of a startup, or building an impressive side project. It can also be shown through a big project in the day job, but that requires a bit more digging.
+- Communication. Do they have spelling errors in their cover letter? What does their online presence look like?
+  - More so than other companies, all of our communication is written and public for the world to see. Good written communication is key.
+
+### Interview with Tim
+
+I tend to split it up into the following sections
+- Talk about PostHog, where were at and what the future looks like, including our long term vision
+- Talk about the candidate, dig into any questions I have from their CV
+- Talk about the hiring process and ask about expected compensation
+
+### Interview with James
+
+-
+
+### Interview with someone from the team
+
+This is up to the interviewer. It can be useful to dig into some technical areas of their work.
+
+### 3 days of contracting work
+
+We offer everyone who's gotten through the first 3 stages 3 days of contracting work. This gives the candidate a chance to learn how we work, and for us to see the quality, speed and communication of the candidate.
+
+If the candidate is currently employed full time and can't take time off, we still offer 3 days but spread out over their evenings or weekends.
+
+We will pay the candidate their 'normal day rate.' If they have done contracting before they will have one, if not you can use [this formula](https://www.ellwoodatfield.com/event/how-to-calculate-out-your-day-rate/) to calculate it.
+
+The candidate will be invited to the team's standups and can pick up work like any other PostHog employee.
+
+### Decision
+
+There will be a written catchup over Slack or via Zoom about the candidate with all people involved during the hiring. A yes/no decision will be made and then communicated to the candidate. 
+
+### After
```

Comment:
This could be stronger for rejected candidates.

Reply:
Done